contents/tutorials/react-ab-testing.md

```diff
@@ -0,0 +1,165 @@
+cd react-flags
+```
+
+Next, remove the boilerplate code in `src/App.js` to simplify it to only a title and button we encourage users to press.
+
+```js
+// src/App.js
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <h1>We are testing this button:</h1>
+      <button>Click me!</button>
+    </div>
+  );
+}
+
+export default App;
+```
+
+Finally, run `npm start` and go to [http://localhost:3000/](http://localhost:3000/) to see our new homepage.
+
+![React app](../images/tutorials/react-ab-testing/app.png)
+
+## Adding PostHog
+
+To use PostHog to manage and track our A/B test, we must install and set it up. We do this by installing the `posthog-js` library to get access to the [React SDK](https://posthog-git-react-feature-flags-post-hog.vercel.app/docs/libraries/react).
+
+```bash
+npm install posthog-js
+```
+
+Once installed, import PostHog into `src/index.js` and set up a client using our project API key and instance address from [our project settings](https://app.posthog.com/project/settings). We can wrap our app in the React `PostHogProvider` to access PostHog in any component.
+
+```js
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+posthog.init('<ph_project_api_key>', {
+  api_host: '<ph_instance_address>',
+})
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <PostHogProvider client={posthog}>
+      <App />
+    </PostHogProvider>
+  </React.StrictMode>
+);
+```
+
+Use the React `usePostHog` hook in our `src/index.js` file to capture a custom event when the button is clicked. This will be the goal metric for our A/B test.
+
+```js
+// src/App.js
+import './App.css';
+import { usePostHog } from posthog-js/react
+
+function App() {
+	const posthog = usePostHog()
+
+const handleClick = () => {
+    posthog.capture('button clicked');
+  };
+
+  return (
+    <div className="App">
+      <h1>We are testing this button:</h1>
+      <button onClick={handleClick}>Click me!</button>
+    </div>
+  );
+}
+
+export default App;
+```
+
+Once done, go back to your app, click the button, and you should see a `button clicked` event captured into your PostHog instance.
+
+![Button clicked event](../images/tutorials/react-ab-testing/event.png)
+
+## Creating an experiment
+
+With PostHog set up, your React app is ready for the A/B test. To create one, go to the [experiments tab](https://app.posthog.com/experiments) in PostHog and click "New experiment." Add a name, feature flag key (like `home-button-test`), set your goal type to a trend of the `button clicked` event, and press "Save as draft." 
+
+![A/B test set up](../images/tutorials/react-ab-testing/ab-test.png)
+
+You can customize it further with a description, secondary metrics, or more variants (for [A/B/n testing](/tutorials/abn-testing)). 
```

Comment:
I think multivariate testing may be a more common phrase

```suggestion
You can customize it further with a description, secondary metrics, or more variants (for [multivariate testing](/tutorials/abn-testing)). 
```

Reply:
based on google search, you're right 😄